### PR TITLE
Add base, avax, optimism migration

### PR DESCRIPTION
**Pull request by ianrowan**
## Link to Issue
Closes: #5675 
## Description of Changes
- Adds migration to add three new chains
- Will be included in create community and groups now

## Test Plan
- Run migration
- Confirm base, avax, optimism are available in create community and groups
- confirm group/community can be created
- 
## Deployment Plan

1. run migration
2. New alchemy app to be created with `private_url` updated post merge

## Other Considerations

- Alchemy has nodes for base and optimism, but not avax so used an avax public node. Could have rate limits etc

### CODE REVIEW

**Review by jnaviask — APPROVED** [review]

**Comment by jnaviask** [comment]
@ianrowan we should keep an eye on usage and add whitelist + private key if we start seeing spikes that aren't ours.

**Comment by timolegros** [comment]
> @ianrowan we should keep an eye on usage and add whitelist + private key if we start seeing spikes that aren't ours, as this PR technically revealed our keys (though they'd be revealed on the site anyway).

We should probably just do what we do for all the others and whitelist these now + create new apps with private keys that we can manually update in `private_url` once this PR is deployed.

File: packages/commonwealth/server/migrations/20240110174904-avax-base-optimism-chainNodes.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    const avaxChainNode = {
+      url: 'https://api.avax.network/ext/bc/C/rpc',
+      eth_chain_id: 43114,
+      alt_wallet_url: 'https://api.avax.network/ext/bc/C/rpc',
+      balance_type: 'ethereum',
+      name: 'Avalanche',
+    };
+
+    queryInterface.bulkInsert('ChainNodes', [
```
Comment (timolegros) [inline packages/commonwealth/server/migrations/20240110174904-avax-base-optimism-chainNodes.js:31]: Not entirely sure if it matters for the migration but we should await or return here this function call to be safe.

**Review by timolegros — CHANGES_REQUESTED** [review]
Small change. Also looking for deployment plan to be updated with note to whitelist + manually update `private_url`.

File: packages/commonwealth/server/migrations/20240110174904-avax-base-optimism-chainNodes.js (hunk 1 above)
Comment (ianrowan) [reply packages/commonwealth/server/migrations/20240110174904-avax-base-optimism-chainNodes.js]: fixed this

**Comment by ianrowan** [comment]
@jnaviask @timolegros Yeah, didn't think it would be an issue exposing the api urls as they are visible frontend. However would there have been an alternate route here?(I just mirrored the setup of our ETH rpc which has ETH alchemy app as url/alt url) 
And what do we mean by whitelist, whitelist on alchemy?

**Comment by timolegros** [comment]
> @jnaviask @timolegros Yeah, didn't think it would be an issue exposing the api urls as they are visible frontend. However would there have been an alternate route here?(I just mirrored the setup of our ETH rpc which has ETH alchemy app as url/alt url) And what do we mean by whitelist, whitelist on alchemy?

Check out the `App Details` of CW Ethereum 2 app on Alchemy. We whitelisted all relevant domains. We should do the same thing for any other public-facing apps.

No alternate route except maybe whitelist before publishing the tokens publicly but we've done that accidentally many times now so no worries.

**Review by timolegros — APPROVED** [review]

**Pull request merged**